src/Aspire.Hosting/Dcp/DcpHostService.cs

```diff
@@ -51,7 +51,7 @@ internal sealed partial class DcpHostService : IHostedLifecycleService, IAsyncDi
 
     public async Task StartAsync(CancellationToken cancellationToken = default)
     {
-        if (_publishingOptions.Publisher is not null && _publishingOptions.Publisher != "dcp")
+        if (_publishingOptions.Publisher is not null and not "dcp")
```

Comment:
is there a reason we have a hardcoded string here?

Reply:
I've been planning on introducing `KnownPublishers` for the string. It appears in many places throughout the code.

De-duplicating the condition makes sense.

Reply:
https://github.com/dotnet/aspire/pull/1972